packages/thegraph-data-access/src/queries.ts

```diff
@@ -108,3 +106,14 @@ export const GetTransactionByDataHashQuery = gql`
     }
   }
 `;
+
+export const GetTransactionsByTopicsCount = gql`
+  query GetTransactionsByTopicsCount($topics: [String!]!) {
+    ${metaQueryBody}
+    channels(where: { topics_contains: $topics }) {
+      transactions {
+        id
+      }
+    }
+  }
+`;
```

Comment:
What is this?

Reply:
I reverted some code I believe it came with it. Removing it